### DISCUSSION TITLE
Docker Start "mc-server" but on a different version?

### DISCUSSION THREAD

**Discussion by ImagineZA** [itzg/docker-minecraft-server#2218] [Q&A]
Is it possible to change the version the MC server runs on when doing the start command, or do I have to rebuild a new container each time I want to change the version?

Ran into an issues where mods weren't ready for 1.20.1 and the server won't start as it wants to run on 1.20.
So a simple docker start VERSION=1.20 would help here.

**Comment by itzg** [comment]
Any changes to environment variables requires recreating the container. I highly recommend using docker compose since it manages that re-create operation automatically.
